### PR TITLE
Platform utils

### PR DESCRIPTION
**Pull request by anondev2323**
- removes `parseAddress` from platforms since it is the same functionality as `toNative`
- separates generic (i.e. non-wormhole specific) platform utils into their own module
- adds native decimals and `getCurrentBlock`
- Cleans up `get.../mustGet` contract access methods. Per an earlier discussion, it's preferred to just throw an error

TODO:
- [ ] getBalance should assume the token address is on the correct chain.  Therefore it should accept a token address instead of TokenId/native
- [x] remove parseAddress tests

### CODE REVIEW

**Comment by barnjamin** [comment]
> Cleans up get.../mustGet contract access methods. Per an earlier discussion, it's preferred to just throw an error
YEESSSSS

File: core/definitions/src/testing/mocks/platform.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export function mockPlatformFactory<P extends PlatformName>(
 export class MockPlatform<P extends PlatformName> implements Platform<P> {
   // @ts-ignore
   readonly platform: P;
+  readonly nativeDecimals = 18n;
```
Comment (barnjamin) [inline core/definitions/src/testing/mocks/platform.ts:43]: can we get this from consts + platform?

It looks like we specify decimals by chain here, maybe it should be platform?
https://github.com/wormhole-foundation/connect-sdk/blob/develop/core/base/src/constants/decimals.ts#L18

File: platforms/evm/src/platform.ts [hunk 2]
```diff
@@ -43,7 +36,18 @@ export module EvmPlatform {
...
+  export const {
+    nativeTokenId,
+    isNativeTokenId,
+    isSupportedChain,
+    getDecimals,
+    getBalance,
+    sendWait,
+    getCurrentBlock,
+    chainFromRpc,
+  } = EvmUtils;
```
Comment (anondev2323) [inline platforms/evm/src/platform.ts:50]: Do you know how to just re-export all of these? Couldn't figure out the right syntax :\

File: platforms/solana/src/address.ts [hunk 3]
```diff
@@ -17,6 +17,9 @@ declare global {
   }
 }
 
+// TODO: is there a zero address for Solana?
+export const SolanaZeroAddress = '11111111111111111111111111111111';
```
Comment (anondev2323) [inline platforms/solana/src/address.ts:21]: TODO

**Review by barnjamin — APPROVED** [review]

**Pull request merged**